providers/base/units/mtd/jobs.pxu

```diff
@@ -0,0 +1,42 @@
+id: mtd/check-total-numbers
+category_id: mtd
+_summary: Check the total MTD numbers for the platform
+_description:
+  Check the total MTD numbers for the platform, relies on the user specifying the number of mtd. 
+  Usage of parameter: {numbers}
```

Comment:
This job is not parameterized and this may be misleading. If you want to express that the job uses environment variable $TOTAL_MTD_NUM to specify what's the right number of MTD devices.

Also there's no environ section in this job, so that variable would be lost when checkbox would elevate the privileges.

Reply:
HI @kissiel,
I revise the description and add the `environ: TOTAL_MTD_COUNT` to mtd/check-total-count job, please help review it again. Many thanks.